dags/quicksuggest2bq.py

```diff
@@ -0,0 +1,45 @@
+"""
+Runs a Docker image that imports Quicksuggest suggestions
+from Remote Settings to BigQuery.
+
+The container is defined in
+[docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/quicksuggest2bq)
+"""
+
+from airflow import DAG
+from datetime import datetime, timedelta
+from utils.gcp import gke_command
+
+
+default_args = {
+    "owner": "aplacitelli@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2021, 11, 18),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+project_id = "TODO"
+table_id = "TODO"
```

Comment:
@jklukas what should we put here?

Reply:
Let's make this `moz-fx-data-shared-prod` and `search_terms_derived.remotesettings_suggestions_v1`.